alpha/declcfg/load.go

```diff
@@ -22,13 +22,74 @@ const (
 	indexIgnoreFilename = ".indexignore"
 )
 
+type WalkMetasFSFunc func(path string, meta *Meta, err error) error
+
+func WalkMetasFS(root fs.FS, walkFn WalkMetasFSFunc) error {
+	return walkFiles(root, func(root fs.FS, path string, err error) error {
+		if err != nil {
+			return walkFn(path, nil, err)
+		}
+
+		f, err := root.Open(path)
+		if err != nil {
+			return walkFn(path, nil, err)
+		}
+		defer f.Close()
+
+		return WalkMetasReader(f, func(meta *Meta, err error) error {
+			return walkFn(path, meta, err)
+		})
+	})
+}
+
+type WalkMetasReaderFunc func(meta *Meta, err error) error
+
+func WalkMetasReader(r io.Reader, walkFn WalkMetasReaderFunc) error {
+	dec := yaml.NewYAMLOrJSONDecoder(r, 4096)
+	for {
+		doc := json.RawMessage{}
+		if err := dec.Decode(&doc); err != nil {
+			if errors.Is(err, io.EOF) {
+				break
+			}
+			return walkFn(nil, err)
+		}
+		doc = []byte(strings.NewReplacer(`\u003c`, "<", `\u003e`, ">", `\u0026`, "&").Replace(string(doc)))
```

Comment:
I don't see a `bytes.NewReplacer`. There _is_ `bytes.Replace(All)`, but as far as I can tell, they only handle one substitution at a time. So I think options are:
1. Roll with this and tolerate the double-allocation
2. Call `bytes.Replace` in a loop for each replacement. (seems worse from an allocation standpoint since we have 3 replacements)
3. Roll our own bytes replacer.
4. Figure out a way to not require this replacement in the first place. It's been awhile, but iirc, its because an encoder is used somewhere along the line that we don't control and therefore can't set [`enc.SetEscapeHTML(false)`](https://pkg.go.dev/encoding/json#Encoder.SetEscapeHTML)
5. Not bother with the replacement and let the unicode show up in the output.

Reply:
I mis-remembered this, sorry - I used [this library](https://pkg.go.dev/go4.org/bytereplacer) in the past and found it nice.

Reply:
One producer of these artifacts is `opm render` itself. :)  If we output YAML, it escapes html-hostile runes like these, so we have to replace them on ingest.